### DISCUSSION TITLE
Recording Audio through Bluetooth mic and saving it as  .wav in SPIFFS

### DISCUSSION THREAD

**Discussion by ProHarshit** [pschatzmann/ESP32-A2DP#588] [Q&A] [answered]
Is this possible? 
Please provide a sample code if possible.

**Comment by pschatzmann** [comment, marked as answer]
I don't quite understand what you want to do and how this would involve A2DP.
If you are interested in MP3 encoding I suggest to have a look at the [AudioTools](https://github.com/pschatzmann/arduino-audio-tools) project and read the Wiki about Codecs. 

Just a word of warning: mp3 encoding is using a lot of resources and I would not recommend to use it on microcontrollers.

**Reply by ProHarshit** [reply]
I hope it is clear through the title now. I have updated it. 
Also sorry for mentioning MP3, I meant wav only
